bus-mapping/src/rpc.rs

```diff
@@ -199,4 +213,17 @@ mod rpc_tests {
         );
         assert!(!trace_by_hash[0].struct_logs.is_empty());
     }
+
+    #[ignore]
+    #[tokio::test]
+    async fn test_get_contract_code() {
+        let transport = Http::new(Url::parse("http://localhost:8545").unwrap());
+        let prov = GethClient::new(transport);
+        let contract_address =
+            Address::from_slice(b"0xD5F110B3E81dE87F22Fa8c5e668a5Fc541c54E3d");
```

Comment:
Can you confirm if you've this test locally successfully?
The method `Address::from_slice` is not supposed to take a hex string.  For that you can use `Address::from_str` or the `address` macro from `crate::address`.

Reply:
Yeah I fixed.